src/components/Recipes/Recipes.jsx

```diff
@@ -0,0 +1,114 @@
+import List from '@mui/material/List';
+import Divider from '@mui/material/Divider';
+import Typography from '@mui/material/Typography';
+import React, { useEffect, useState } from 'react';
+import * as recipesClient from '../../api/recipes';
+import AddRecipe from '../AddRecipe';
+import Recipe from '../Recipe';
+
+function Recipes() {
+  const [recipes, setRecipes] = useState([]);
+
+  const fetchRecipes = () => {
+    recipesClient
+      .getRecipes()
+      .then(setRecipes);
+  };
+
+  useEffect(fetchRecipes, []);
+
+  const addRecipe = ({ name, description, ingredients = [] }) => {
+    recipesClient
+      .addRecipe({ name, description, ingredients })
+      .then((newRecipe) => {
+        setRecipes([...recipes, newRecipe]);
```

Comment:
you could probably use `fetchRecipes` like you did on `editRecipe` here

Reply:
Yeah good suggestion!